scylla-server/src/services/driver.services.ts

```diff
@@ -0,0 +1,27 @@
+import prisma from '../prisma/prisma-client';
+import { ResponseFunction } from '../utils/message-maps.utils';
+
+/**
+ * CRUD operation to get all dataTypes
+ * @returns string containing all the dataTypes
+ */
+export const getAllDrivers: ResponseFunction = async () => {
+  const data = await prisma.driver.findMany();
+  return JSON.stringify(data);
+};
+
+/**
+ * CRUD operation to create a driver in the database if it doesn't already exist, does nothing otherwise.
+ *
+ */
+export const upsertDriver = async (driver_name: string) => {
```

Comment:
we dont user _ really, should be camel case

Reply:
I see i see changing it rn.